WordPress/src/main/res/drawable/ic_briefcase_grey_24dp.xml

```diff
@@ -0,0 +1,13 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="24dp"
+    android:width="24dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0" >
+
+    <path
+        android:fillColor="@color/grey_text_min"
```

Comment:
Other icons that use `grey_text_min` are named `ic_<name>_grey_min_24dp.xml`.  Can we rename this to `ic_briefcase_grey_min_24dp.xml` to be consistent?